osbs/conf.py

```diff
@@ -362,6 +363,15 @@ def get_build_imagestream(self):
     def get_proxy(self):
         return self._get_value("yum_proxy", self.conf_section, "yum_proxy")
 
+    def get_labels(self):
+        labels_list = self._get_value("labels", self.conf_section, "labels")
+        if labels_list is None:
+            return {}
+        # When read from config file, it needs to be deserialized
+        if type(labels_list) is not list:
+            labels_list = json.loads(labels_list)
```

Comment:
Does this mean the config syntax is something like `labels = [("label1", "value1"), ("label2", "value2"), ("label3", "value3")]`?

Looks pretty weird to me:) But then again, we probably don't expect users to set labels in the configuration, do we? Should we just not allow setting this in the config?